.github/workflows/release.yml

```
@@ -30,6 +30,15 @@ jobs:
       - uses: fregante/daily-version-action@v1
         name: Create tag if necessary
         id: daily-version
+      - name: Create release
+        uses: actions/github-script@v4.0.2
```

Let's use the newest version directly.

```suggestion
        uses: actions/github-script@v5.0.0
```